games-arcade/vvvvvv/vvvvvv-2.3~git.recipe

```diff
@@ -10,10 +10,10 @@ purchasing a copy at https://thelettervsixtim.es."
 HOMEPAGE="https://thelettervsixtim.es"
 COPYRIGHT="Terry Cavanagh 2010-2020"
 LICENSE="VVVVVV Source Code License v1.0"
-REVISION="5"
-srcGitRev="5fdbaa007608e11d91a81dda7f0523255572018e"
+REVISION="6"
```

Comment:
Revision should be "1" here as this is a version bump

Reply:
The package has been updated multiple times in the past, mostly due to keeping up with changes upstream. 2.3 is the first known instance of a formal version bump, yet new fixes and features came well before it.

Reply:
Last official release is version 2.2, as this bumps to a git checkout to a "newer" version REVISION should be reset to mark a difference between last official version and a new release in progress

Reply:
Ah, ok then. My prior understanding of REVISION was a way for the package manager to determine whether a package is newer than the locally installed one.

Reply:
If no 2.2 version was released then yes, but there is an official tarbal for it, development went on after 2.2 preparing it for version 2.3, so a revision change would be needed

Reply:
Hold on :)

Reply:
Did you check if this change was imported in the official release (the commit is older than the official release)

Reply:
This was from 2 days ago while 2.3 was in development. Should I update it to the current commit presently?

Reply:
Looking at their github page they announced a 2.2 release 18hours ago? 
https://github.com/TerryCavanagh/VVVVVV/releases/tag/2.2

Reply:
If so then yes, bump to the latest commit there to keep version 2.3

Reply:
The commit which this is based on is on 2.3 work. One of the maintainers mentions this in the issue linked in the PR description:
> I'd be up for disabling the option in Haiku builds for 2.3

Do I still need to update the recipe to the lates upstream commit?

Reply:
Ok, if this is working for you then keep the current upstream commit and version 2.3, only revision should still be changed then